app-emulation/vagrant/vagrant-2.2.14.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${RDEPEND}
```

Comment:
Is this some ruby-related thing, or a typo :P 
I see you're calling `ruby_add_rdepend` below so this looks a bit weird to me.

Reply:
As far as I know RDEPEND doesn't stack so this is meant to keep the (non ruby_add_rdepend) dependencies such as dev-lang/ruby rather than overwrite them.